tests/test_clusterv2.py

```diff
@@ -384,4 +384,54 @@ def test_minimal(self):
         Connection._api_call = Mock(return_value={})
         qds.main()
         Connection._api_call.assert_called_with('POST', 'clusters/1234/clone', {'cluster_info':
-                                                                                    {'label': ['test_label1', 'test_label2']}})
\ No newline at end of file
+                                                                                    {'label': ['test_label1', 'test_label2']}})
+
+class TestClusterList(QdsCliTestCase):
```

Comment:
@abhijitjaiswal : These test cases are not similar to what we have in test_cluster.py. Some of the differences are:

- Mock is not initialised properly when `--state` is passed in arguments.

- `test_state_invalid()` expects the command line to fail with `SystemExit` because an `invalid` state is passed in `--state` argument. 

Assuming that these tests are passing suggests that the `list` cluster behaviour in v1.2 and v2 is not same which should not be the desired behaviour. Can you please look into it again ?